sklearn/mixture/gaussian_mixture.py

```diff
@@ -648,3 +686,32 @@ def aic(self, X):
             The greater the better.
         """
         return -2 * self.score(X) * X.shape[0] + 2 * self._n_parameters()
+
+    @property
+    def precisions_(self):
+        self._check_is_fitted()
+        if self.covariance_type is 'full':
+            return np.array([np.dot(prec, prec.T)
+                            for prec in self.precisions_cholesky_])
+        elif self.covariance_type is 'tied':
+            return np.dot(self.precisions_cholesky_,
+                          self.precisions_cholesky_.T)
+        else:
+            return self.precisions_cholesky_ ** 2
```

Comment:
not sure I like this, meaning properties/attributes that can trigger a lot of computation. It can make code slow for weird reasons.

Do we really need this attributes?

I think I'd actually prefer to precompute them even if it's redundant.